examples/sockeye_translate/README.md

```diff
@@ -0,0 +1,71 @@
+# sockeye-serving
+This example shows how to serve Sockeye models for machine translation.
+The custom handler is implemented in `sockeye_service.py`.
+Since Sockeye has many dependencies, it's convenient to use Docker.
+For simplicity, we'll use a pre-trained model and make some assumptions about how we preprocess the data.
+
+## Getting Started With Docker
+Pull the latest Docker image:
+```bash
+docker pull jwoo11/sockeye-serving
+```
+
+Download the example model archive file (MAR).
+This is a ZIP archive containing the parameter files and scripts needed to run translation for a particular language:
+* https://www.dropbox.com/s/pk7hmp7a5zjcfcj/zh.mar?dl=0
+
+Extract the MAR file to `/tmp/models`.
+ We'll use this directory as a bind mount for Docker:
+```bash
+unzip -d /tmp/models/zh zh.mar
```

Comment:
Just want to let you know, unzip .mar is not required.
You can put .mar directly in /tmp/models folder.

Reply:
My custom handler references scripts in the unzipped directory. Would it still work to use the MAR directly?

Reply:
Before creating mar file, with model-archiver tool, put all the files required for your model in the same "model-path" folder. If all the files are in the .mar file, then you would be able to refer to those files because the unzipped directory is in the PYTHONPATH.

Reply:
@jamesewoo 
MMS will unzip .mar file into temp folder. It should work for your case as long as your custom handler properly handle the relative file path.
The can use two ways to get your model dir:
1. context.system_properties.get("model_dir") -- recommend
2. os.getcwd()

Reply:
model-server is the home directory for username "model-server". What are you trying to do with the .pl file?

Reply:
I just got it working with only the MAR file. Working on updating the README. The .pl files are for pre and post-processing the text. They run as subprocesses in the custom handler.

Reply:
Local MAR and extracted MAR work fine, but I can't seem to get a remote MAR working. I'm passing in "https://www.dropbox.com/s/pk7hmp7a5zjcfcj/zh.mar?dl=1" as the URL and I get 
`{
  "code": 404,
  "type": "ModelNotFoundException",
  "message": "Load failed... Deregistered model zh"
}`

Reply:
This is interesting. It seems to work. Do you see any other error in the logs? The model is downloadable, so that isn't the issue. You could look at mms_log.log to get more info. Could you share that as well.

Reply:
Paste the whole error log. Not just the Java exception, the backend worker exceptions run longer than the Java exceptions :) . If possible just load the file to your dropbox account, I can try to help you with it.

Reply:
I got it working based on your Dropbox comment. I just needed to update the MAR file in my Dropbox, whoops!